db/seeds.rb

```diff
@@ -167,6 +167,27 @@
       is_optional: false,
       is_repeatable: false,
     ),
+    Page.create(
+      question_text: "Are you eligible to submit this form?",
+      answer_type: "selection",
+      answer_settings: {
+        only_one_option: "true",
+        selection_options: [
+          { "name": "Yes" },
+          { "name": "No" },
+        ],
+      },
+      is_optional: false,
+    ),
```

Comment:
One minor note, I think you'd normally put the exit page at the beginning of a form! 😁

Reply:
Thats a good point, I was trying to avoid having to change all the page ids :)

Reply:
It was actually less work to put it at the start 🙄 

I've moved it and removed the extra question.